packages/seacas/libraries/ioss/src/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ IF (${PACKAGE_NAME}_ENABLE_Pamgen)
   ADD_SUBDIRECTORY(pamgen)
 ENDIF()
 
+IF (${PACKAGE_NAME}_ENABLE_DataWarehouse)
```

Comment:
Not clear if this should be TPL_ENABLE_DataWarehouse or ${PACKAGE_NAME}_ENABLE_DataWarehouse.  I think I have been using the `package_name` form if the library is in Trilinos and the `TPL` version if the library is external from SEACAS and Trilinos.